src/quo/components/utilities/social/loader.cljs

```diff
@@ -0,0 +1,14 @@
+(ns quo.components.utilities.social.loader
+  (:require-macros [quo.components.utilities.social.loader :as loader])
+  (:require [taoensso.timbre :as log]))
+
+(def socials (loader/resolve-socials))
```

Comment:
Please mark it as privarte, just to ensure this is not used from an external source :)

Reply:
Done